datafusion/tests/test_functions.py

```diff
@@ -25,6 +25,8 @@
 from datafusion import functions as f
 from datafusion import literal
 
+np.seterr(invalid="ignore")
```

Comment:
I think it's reasonable to turn off warnings of invalid floating-point errors, which can safely be ignored in test cases.